examples/siro_sandbox/README.md

```diff
@@ -8,19 +8,31 @@ This is a 3D interactive GUI app for testing various pieces of SIRo, e.g. rearra
 ## Known Issues
 * The policy-driven agent doesn't seem to be working in terms of producing interesting actions. As a placeholder, I've injected random-base-movement behavior in `BaselinesController.act`; see comment "temp do random base actions".
 * One-time visual flicker shortly after app startup on Mac
+* Spot robot stops and doesn't move once it collides with any object (try pressing `M` to reset to a next episode)
 
-## Running HITL eval with a user-controlled humanoid and policy-driven Fetch
+## Running HITL eval with a user-controlled humanoid and policy-driven Fetch or Spot
 
 1. Make sure you've followed the [SIRo install instructions](../../SIRO_README.md#installation).
-2. Run this command:
+2. To use Fetch run:
 ```
 HABITAT_SIM_LOG=warning MAGNUM_LOG=warning \
 python examples/siro_sandbox/sandbox_app.py \
 --disable-inverse-kinematics \
---humanoid-user-agent --never-end \
+--humanoid-user-agent \
+--never-end \
 --cfg benchmark/rearrange/rearrange_easy_human_and_fetch.yaml \
 --cfg-opts habitat.dataset.split=minival
 ```
+3. To use Spot run:
```

Comment:
Do we want to mention that we need to use the latest habitat_sim? thank you!

Reply:
I think this is covered in "SIRo install instructions" mentioned in step 1.